### PR TITLE
Lesson9

### PR DESCRIPTION
**Pull request by alex-rns**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/BelodubrovskyiAn/CustomWidget/view/frontend/templates/widget/lesson9_banner.phtml [hunk 1]
```diff
@@ -1,6 +1,15 @@
-<?php if ($block->getData('title')): ?>
-    <button class="title"
-            onclick="location.href='/geekhub-cms'">
-        <?php echo $block->getData('title'); ?>
-    </button>
-<?php endif; ?>
+<div class="geekhub-custom-widget">
+    <?php if ($block->getData('block_id')) : ?>
```
Comment (APanteleychuk) [inline app/code/BelodubrovskyiAn/CustomWidget/view/frontend/templates/widget/lesson9_banner.phtml]: Please use set/get methods getTitle() etc.

File: app/code/BelodubrovskyiAn/CustomWidget/view/frontend/templates/widget/lesson9_banner.phtml [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+        <?php echo $block->getLayout()
```
Comment (APanteleychuk) [inline app/code/BelodubrovskyiAn/CustomWidget/view/frontend/templates/widget/lesson9_banner.phtml]: Needs to be moved in block like function.

**Review by APanteleychuk — CHANGES_REQUESTED** [review]
Need to fix

File: app/code/BelodubrovskyiAn/CustomWidget/view/frontend/templates/widget/lesson9_banner.phtml (hunk 1 above)
Comment (alex-rns) [reply app/code/BelodubrovskyiAn/CustomWidget/view/frontend/templates/widget/lesson9_banner.phtml]: Fixed.

**Pull request merged**